### ISSUE TITLE
[BUG] Project picture not working

### ISSUE THREAD

**Issue by qaniere** [Leantime/leantime#1626]
**What set up are you using**
* [ ] Cloud Hosted Version
* [ X] Self Hosted

**Describe the bug**
Tried to upload a projet picture, clicked submit, page reloaded and nothing happend

**To Reproduce**
1. Upload a project picture

**Expected behavior**
Be able to upload a projet picture

**Leantime Version**
docker:latest (2.3.21 i belive)

**Server**
Idk, i'm using docker

**PHP / MySQL Version**
Idk, i'm using docker

Can help if needed, basic PHP knowledge and strong javascript knowledge, just need to know where to look :)

**Comment by cbrand-ludlums** [comment]
I'm also using docker and the avatar upload worked it just doesn't update in the browser.  The project or user avatar didn't show up until I did a refresh in the browser by holding down Shift+Refresh then they showed up correctly.  The user avatar worked fine until the 2.3.21 update.  Maybe something with the "Improved profile image and project avatar image caching" from the release notes is causing an issue?

**Comment by qaniere** [comment]
Yes it's seems to be working when clearing browser cache. I'll keep the issue opened since something can be implemented to force browser cache to clear ?
